__tests__/testFunc/book.js

```diff
@@ -0,0 +1,409 @@
+            .expect(200);
+    });
+    it('GET: /books, 401, unauthorized', (done) => {
+        request(app)
+            .get('/books')
+            .set('Accept', 'application/json')
+            .expect('Content-Type', /json/)
+            .expect((res) => {
+                expect(res.body.message).toBe('Не авторизировано');
+            })
+            .expect(401, done);
+    });
+    it('GET: /books/users=true, 200, authorized', async () => {
+        const object = [{
+            _id: expect.any(String),
+            title: 'ruhatestbook',
+            authorId: {
+                _id: expect.any(String),
+                login: 'ruhatest',
+                password: '123',
+                isAdmin: false,
+                __v: 0,
+                token: expect.any(String),
+                avarageRating: expect.any(Number),
+            },
+            rating: expect.any(Number),
+            comment: expect.any(Array),
+            __v: 0,
+        }];
+        await request(app)
+            .post('/auth/sign-up')
+            .send({ login: login.fullLogin, password: password.fullPassword });
+        const signToken = await request(app)
+            .post('/auth/sign-in')
+            .send({ login: login.fullLogin, password: password.fullPassword });
+        const signId = await request(app)
+            .get('/users/me')
+            .set('Authorization', signToken.body.token);
+        await request(app)
+            .post('/books')
+            .set('Authorization', signToken.body.token)
+            .send({ title: title.fullTitle, authorId: signId.body._id });
+        await request(app)
+            .get('/books')
+            .query('users=true')
+            .set('Authorization', signToken.body.token)
+            .expect('Content-Type', /json/)
+            .expect((res) => {
+                expect(res.body).toEqual(object);
+            })
+            .expect(200);
+    });
+    it('GET: /books/authorId=existed id, 200, authorized', async () => {
+        await request(app)
+            .post('/auth/sign-up')
+            .send({ login: login.fullLogin, password: password.fullPassword });
+        const signToken = await request(app)
+            .post('/auth/sign-in')
+            .send({ login: login.fullLogin, password: password.fullPassword });
+        const signId = await request(app)
+            .get('/users/me')
+            .set('Authorization', signToken.body.token);
+        await request(app)
+            .get('/books')
+            .query({ authorId: signId.body._id })
+            .set('Authorization', signToken.body.token)
+            .expect('Content-Type', /json/)
+            .expect(200);
+    });
+    it('GET: /books/:id/comments, 200, authorized', async () => {
+        const object = [{
+            text: 'ruhatestcommentget',
+            date: expect.any(String),
+            commentator: expect.any(String),
+        }];
+        await request(app)
+            .post('/auth/sign-up')
```

Comment:
Same here. No need to sign up many times. Do sign up and sign in in beforeAll and use the same user everywhere